### PR TITLE
DDP-6358: Add email reminder to sign stool collection form

### PR DESCRIPTION
**Pull request by alextanlan**
## Context

_Say a few words about the bigger picture beyond this PR. How do your changes improve life for our users? Link directly to the jira ticket (just type "DDP-..." and it'll automatically link) to help your teammates quickly get up to speed on the motivation for these changes._

## Checklist

- [ ] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [ ] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: study-builder/studies/pancan/emails/consent_completed.tmpl.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["instructions"]}</p>
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["provide_samples"]}</p>
+        <p style="font-family: Roboto, sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">${text["complete_process"]}</p>
+      </td>
+    </tr>
+    <tr style="width: 540px;">
+      <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+        <div>&nbsp; <a href="{{baseWebUrl}}/dashboard" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```
Comment (alextanlan) [inline study-builder/studies/pancan/emails/consent_completed.tmpl.html]: Not sure if the link correct for the button "Contact Information Form" ?

File: study-builder/studies/pancan/study-events.conf [hunk 2]
```diff
@@ -119,10 +119,8 @@
 //        "type": "SENDGRID_EMAIL",
 //        "templates": [
 //          {
-//            "emailTemplate": ${emails.joinMailingList},
-//            "language": "en",
-//            "isDynamic": true
-//          }
+//            "emailTemplate": ${emails.en_consent_completed}, "language": "en", "isDynamic": true },
```
Comment (alextanlan) [inline study-builder/studies/pancan/study-events.conf]: Not sure if I applied the email for a proper event.
It is not quite clear from [the DDP-6358 ticket](https://broadinstitute.atlassian.net/browse/DDP-6358).

**Comment by yufengwng** [comment]
I thought the "stool sample" tickets are not a requirement for launch yet?

**Comment by MocanaAtBroad** [comment]
> I thought the "stool sample" tickets are not a requirement for launch yet?
Not required for launch. I should have de-prioritized these tickets earlier.
But no harm. This ticket needed to get done at some point anyways.

**Comment by alextanlan** [comment]
The ticket was just next one in its epic.

File: study-builder/studies/pancan/emails/consent_completed.tmpl.html (hunk 1 above)
Comment (yufengwng) [reply study-builder/studies/pancan/emails/consent_completed.tmpl.html]: For linking to activities, what we usually do is put a link like this:
```
{{baseWebUrl}}/activity-link/{{activityInstanceGuid}}
```
The backend will provide the `activityInstanceGuid` for the "Contact Information Form", and on the frontend there will be a component at path `/activity-link` that will take that path parameter and show the exact activity.

Comment (yufengwng) [reply study-builder/studies/pancan/emails/consent_completed.tmpl.html]: Putting a link to the dashboard is also fine. User will see it listed in their dashboard and can open it themselves. Really just depends on what the study staff prefers.

**Review by yufengwng — DISMISSED** [review]
Seems fine to merge up. Since it's not used yet, we can tweak it when the time comes.

**Review by yufengwng — APPROVED** [review]

**Pull request merged**